node-scripts/spec/pipeline1/resources/default/.ocd3.yaml

```diff
@@ -0,0 +1,8 @@
+version: 1
+
+project:
+  name: bahmniapps
```

Comment:
When a YAML file is provided I think the project section will probably go away to only leave the build and deploy sections.